src/components/DisplayTire.tsx

```diff
@@ -1,11 +1,26 @@
-import {calculateTireHeight} from '../../util/helpers';
-import {tire} from "../../types/tire";
+import { calculateTireHeight } from '../../util/helpers';
+import { MouseEventHandler } from 'react';
+import { Tire } from '../../types/tire';
 
+interface DisplayTire extends Omit<Tire, 'height'> {
```

Comment:
Neat! I wasn't aware of Omit. https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys